test/test.sh

```diff
@@ -22,17 +25,18 @@ do
 		echo "$test_case" >> "$THIS_OUTPUT"
 		echo '----------------------------------' >> "$THIS_OUTPUT"
 		./"$BUILD_EXE_FILE" "$test_case" >> "$THIS_OUTPUT" 2>&1
-		# ./"$TEST_EXE_FILE" "$TEST_EXE_PARAMS" "$test_case" >> "$_OUTPUT" 2>&1
-		echo "$TEST_EXE_FILE $TEST_EXE_PARAMS $test_case"
 		echo '----------------------------------' >> "$THIS_OUTPUT"
 		echo '' >> "$THIS_OUTPUT"
 	;;
 	esac
 done
 
+# Generate correct output and store it in `CORRECT_OUTPUT`
+./"$TEST_EXE_FILE" "$TEST_EXE_PARAMS" >> "$OUTPUT" 2>&1
 
+# Compare `THIS_OUTPUT` and `CORRECT_OUTPUT` and store it in `OUTPUT`
 diff -sy --width=120 "$CORRECT_OUTPUT" "$THIS_OUTPUT" > "$OUTPUT"
 
-rm "$THIS_OUTPUT"
-rm "$CORRECT_OUTPUT"
-less "$OUTPUT"
+# rm "$THIS_OUTPUT"
```

Comment:
Delete comments and make some clean in the Makefile

Reply:
Fixed it.